Dockerfile.ppc64le

```diff
@@ -0,0 +1,44 @@
+ARG quay_expiration=never
+ARG release_tag=0.0.0
+
+# golang:1.17 image created 2021-12-04T00:31:06.02014601Z
+FROM docker.io/golang:1.17 AS builder
+ARG quay_expiration
+ARG release_tag
+
+# Build the preflight binary
+COPY . /go/src/preflight
+WORKDIR /go/src/preflight
+RUN make build RELEASE_TAG=${release_tag}
+
+# ubi8:latest
+FROM registry.access.redhat.com/ubi8/ubi:latest
+ARG quay_expiration
+
+# Define that tags should expire after 1 week. This should not apply to versioned releases.
+LABEL quay.expires-after=${quay_expiration}
+
+# Define versions for dependencies
+ARG OPENSHIFT_CLIENT_VERSION=4.7.19
+ARG OPERATOR_SDK_VERSION=1.14.0
+
+# Add preflight binary
+COPY --from=builder /go/src/preflight/preflight /usr/local/bin/preflight
+
+# Install dependencies
+RUN dnf install -y \
+      bzip2 \
+      gzip \
+      iptables \
+      findutils \
+    && dnf clean all
+
+# Install OpenShift client binary
+RUN curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OPENSHIFT_CLIENT_VERSION}/openshift-client-linux-${OPENSHIFT_CLIENT_VERSION}.tar.gz | tar -xzv -C /usr/local/bin oc
+
+# Install Operator SDK binray
+RUN curl -Lo /usr/local/bin/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk_linux_ppc64le \
```

Comment:
Couldn't we just make an ARCH ARG? Default it to amd64, and override it in the action? That way we don't need an entirely new Dockerfile. And i'm fine with bumping the go version in the main dockerfile.

Reply:
@bcrochet I have made the changes. Please review it!